controllers/validator.go

```diff
@@ -78,6 +81,9 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 	return true, nil
 }
 
+// empty struct to use as map value
+type empty struct{}
```

Comment:
Instead of using a map to emulate a set, it might be better to just use a Set implementation like https://github.com/deckarep/golang-set -- we used that in MTC

Reply:
+1

Reply:
but empty struct has size of 0 bytes.. The set linked will be bigger than 0 bytes.

Reply:
resolved via https://github.com/openshift/oadp-operator/pull/784/commits/06a3c10bad8a107dca1dce76fedf138e2edc7554